choco/VERIFICATION.txt

```diff
@@ -0,0 +1,7 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+This package is published by the NUnit Project itself. The binaries are 
+identical to other package types published by the project, in particular
+the NUnit.ConsoleRunner and NUNit.Console nuget packages.
```

Comment:
Also - NUnit.Console won't contain any binaries. Maybe this could be rephrased something along the lines of "Any binaries will [...] in particular the NUnit.ConsoleRunner NuGet package."